sw/device/rom_exts/manifest.hjson

```diff
@@ -170,5 +179,31 @@
       type: "field",
       size: 32,
     },
+    {
+      // This is modelled as an offset, as we don't want this to have a size,
+      // and if we generate assembly like `rom_ext_manifest.S`, we don't want to
```

Comment:
Could you add description saying that the spec requires 256b (2048) alignment for the interrupt vector?